### DISCUSSION TITLE
Is there a way to configure `:separateMultipleMinor`?

### DISCUSSION THREAD

**Discussion by karfau** [renovatebot/renovate#10628] [Request Help] [answered]
There is a configuration preset for separating multiple major versions into dseparate PRs.
Since some tools do not adhere to semver and publish breaking changes with minor versions (looking at you typescript), I would love to configure the same for minor versions.

Is that already possible using package rules?

**Comment by rarkins** [comment, marked as answer]
I don't think there's any way to achieve it using existing config so please propose it as a feature request. separateMultipleMinor sounds ok

**Reply by karfau** [reply]
~https://github.com/renovatebot/renovate/issues/10629~ (Duplicate)
#6576
